libcontainer/integration/init_test.go

```diff
@@ -19,12 +23,40 @@ func init() {
 	}
 	runtime.GOMAXPROCS(1)
 	runtime.LockOSThread()
+
+	logLevel := logrus.InfoLevel
+	if level := os.Getenv("_LIBCONTAINER_LOGLEVEL"); level != "" {
```

Comment:
This was made optional because most (or all) of the test cases do not set Process.LogLevel and it is easier to have the default here rather than to amend all the tests.

Alternatively, the default can be moved somewhere to Process.Start